src/store/flags.ts

```diff
@@ -105,6 +113,16 @@ export const useFlagsSelector: TypedUseSelectorHook<FlagStore> =
 export const useFlagsDispatch: () => FlagDispatch =
   createDispatchHook<FlagStore>(flagsContext);
 
+export function setAnswer({trialId, status, provenanceGraph, answers} : {trialId: string, status: boolean, provenanceGraph?: ProvenanceGraph<any, any, any>, answers: Record<string, any>}) {
```

Comment:
Here is that same ugly type. Let's move this into types.ts and import it in both places

Reply:
Fixed up types